tasks/collect-data/tests/test-collect-data.yaml

```diff
@@ -51,21 +51,15 @@ spec:
               spec:
                 application: foo
                 origin: foo
-                releaseStrategy: foo
-              EOF
-              kubectl apply -f releaseplanadmission
-
-              cat > releasestrategy << EOF
-              apiVersion: appstudio.redhat.com/v1alpha1
-              kind: ReleaseStrategy
-              metadata:
-                name: releasestrategy-sample
-                namespace: default
-              spec:
-                pipeline: foo
                 policy: foo
+                pipelineRef:
+                  resolver: bundles
+                  params:
```

Comment:
these are not the right params for a bundle resolver, they are for a cluster resolver

Reply:
Moved to cluster.